files/ko/web/api/web_audio_api/advanced_techniques/index.html

```diff
@@ -437,56 +437,56 @@ <h3 id="The_final_playSample_function">The final playSample() function</h3>
 }</pre>
 
 <div class="note">
-<p><strong>Note</strong>: The sound file was <a href="http://soundbible.com/1573-DTMF-Tones.html">sourced from soundbible.com</a>.</p>
+<p><strong>참고</strong>: 이 사운드 파일은 <a href="https://soundbible.com/1573-DTMF-Tones.html">soundbible.com에서 찾아졌습니다</a>.</p>
```

Comment:
```suggestion
<p><strong>참고</strong>: 이 사운드 파일의 출처는 <a href="https://soundbible.com/1573-DTMF-Tones.html">soundbible.com</a>입니다.</p>
```
이것도 조금 더 이해하기 쉽도록 의역했습니다!